action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Crashtest Security Action'
+description: 'A Github Action for Running a Crashtest Security scan to perform Dynamic Application Security Testing (DAST).'
+inputs:
+  crashtest-webhook:
+    description: 'Webhook Secret of the Crashtest Security Scan Target'
+    required: true
+  pull-report:
+    description: 'Flag whether the report should be downloaded as JUnit XML file'
+    required: false
+    default: 'false'
```

Comment:
The default is intentionally a string value?

Reply:
Yes. I was not able to add this as a boolean. It seems that only string values are supported here.